action-server/covidflow/utils/screening_locations.py

```diff
@@ -0,0 +1,66 @@
+from typing import Iterable
+
+import backoff
+import structlog
+from aiohttp import ClientError, ClientSession
+
+logger = structlog.get_logger()
+
+CLINIA_ENDPOINT = "https://covid.clinia.com"
+CLINIA_API_ROUTE = "/api/v1/indexes/covid/query"
+
+LOCATION_KEY = "aroundLatLng"
+PAGE_KEY = "page"
+HITS_KEY = "hits"
+
+DEFAULT_SEARCH_PARAMETERS = {
+    "perPage": 20,
+    "rankingInfo": True,
+    "facetFilters": ["services.en:COVID-19 screening"],
```

Comment:
We should confirm with them that locations marked "screening" will contain all locations we need, and verify if we need to also include "testing" locations in case they add some that don't have the "screening" tag

Reply:
I'll ask them for the description of the services. From what i've seen, testing does not seem like something we need to include.

Reply:
Great, I _can_ actually "OR" the items in `facetFilters`.

```
// property1:value AND property2:value
facetFilters: ["property1:value", "property2:value"]
// property1:value OR property2:value
facetFilters: [["property1:value", "property2:value"]]
// (property1:value OR property2:value) AND property3:value
facetFilters: [["property1:value", "property2:value"], "property3:value"]
```